docs/maya/skin_merge.md

```diff
@@ -0,0 +1,66 @@
+# AdnSkinMerge
+
+AdnSkinDeformer is a Maya deformer to merge animation geometries with the results provided by AdnSkin. This deformer allows for the merging of several animation and simulation meshes into a single final mesh.
+
+The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blends weight map.
+
+### How To Use
+
+AdnSkinMerge makes use of its own User Interface to create and modify the deformer.
+
+To create an AdnSkinMerge deformer within a Maya scene, the following inputs must be provided:
+
+  - **Final Mesh (F)**: Mesh to apply the merge results.
+  - **Animation Mesh List (Al)**: Mesh(es) to drive the simulation skin.
+  - **Simulation Mesh List (Sl)**: Mesh(es) with AdnSkin deformer applied.
```

Comment:
Same question as before, could the user provide a mesh which has already the results of the skin simulation?

Reply:
Yes! Applied your suggestion.